daemon/src/projection.rs

```diff
@@ -502,7 +501,9 @@ impl Cfd {
         let latest_price = match Price::new(latest_price) {
             Ok(latest_price) => latest_price,
             Err(e) => {
-                tracing::warn!("Failed to compute profit/loss because latest price is invalid: {e}");
+                tracing::warn!(
+                    "Failed to compute profit/loss because latest price is invalid: {e}"
+                );
```

Comment:
❓ Why do these diffs keep appearing? As in nothing has changed, but these lines are being reformatted.

Reply:
I don't know but I will run the formatter again on each commit to check!